tests/e2e/support.py

```diff
@@ -24,6 +24,14 @@ def create_namespace(self):
     self.api_instance.create_namespace(namespace_body)
 
 
+def create_namespace_with_name(self, namespace_name):
+    self.namespace = namespace_name
+    namespace_body = client.V1Namespace(
+        metadata=client.V1ObjectMeta(name=self.namespace)
+    )
+    self.api_instance.create_namespace(namespace_body)
```

Comment:
We could do with an exception here in case it is not possible to create the namespace

Reply:
Thank you @Bobbins228 for reviewing
Yes, these Nit-pick changes seems valid, I will make chnages accordingly.